var/spack/repos/builtin/packages/py-pybind11/package.py

```diff
@@ -42,10 +44,12 @@ class PyPybind11(CMakePackage, PythonPackage):
     version('2.1.1', sha256='f2c6874f1ea5b4ad4ffffe352413f7d2cd1a49f9050940805c2a082348621540')
     version('2.1.0', sha256='2860f2b8d0c9f65f0698289a161385f59d099b7ead1bf64e8993c486f2b93ee0')
 
-    depends_on('py-setuptools', type='build')
+    depends_on('ninja', type='build')
```

Comment:
I think we want deps on py-cmake and py-ninja. Not sure if it actually makes any difference, but that's what pip would check for

Reply:
We just need `cmake` and `ninja` on our paths, no need to pull them in via `pypa` packages.

Reply:
My only concern is what will happen if pip doesn't find those deps. I _think_ that it will be fine since we use `--no-deps`, but I wonder what would happen if we moved from pip to build/installer which doesn't have that option.

Reply:
Correct, it will ignore them with `--no-deps`.

Reply:
No, it will check build dependencies, but it has

Reply:
[installer](https://pypi.org/project/installer/) will not look at build dependencies: https://installer.readthedocs.io/en/latest/concepts/

I wonder how you would skip dependencies there, but it's not related to the problem here :)